### DISCUSSION TITLE
Support needed for installing exiftool after using script for photoprism installation

### DISCUSSION THREAD

**Discussion by oa01** [tteck/Proxmox#362] [General]
So i installed photoprism using your script, but fail to enable exiftool.
I suppose Exiftools are not installed, could someone help me installing them?
since idk where to install exiftools

**Comment by tteck** [comment]
Try `apt update && apt install exiftool -y` in the LXC console

**Reply by tteck** [reply]
I added exiftool to the `photoprism-install.sh`

**Reply by oa01** [reply]
Thank you, also can you tell me how i add photoprism as a network drive?
Using the steps in the Docs does not work

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/issues/242
